apps/nextjs-website/src/components/molecules/SpeakerPreview/SpeakerPreview.tsx

```diff
@@ -56,58 +63,86 @@ const SpeakerInfo = ({
   name,
   jobTitle,
   description,
+  isSmallScreen = false,
 }: SpeakerInfoProps) => {
   const { palette } = useTheme();
   return (
     <Box
       sx={{
+        maxHeight: '5rem',
+        maxWidth: '15.625rem',
         display: 'flex',
         flexDirection: 'column',
         justifyContent: 'center',
+        gap: '0.375rem',
       }}
     >
       {compactMode ? (
         <>
-          <Typography variant='body2' fontWeight={600}>
+          <Typography fontSize={'1.125rem'} fontWeight={600}>
             {name}
           </Typography>
-          <Typography variant='body2'>{jobTitle}</Typography>
+          <Typography fontSize={'1.125rem'} fontWeight={400}>
+            {jobTitle}
+          </Typography>
         </>
       ) : (
         <>
-          <Typography variant='h5' fontWeight={700}>
+          <Typography
+            fontWeight={700}
+            fontSize={{ xs: '1.375rem', md: '1.5rem' }}
+            lineHeight={{ xs: '1.813rem', md: '2rem' }}
+          >
             {name}
           </Typography>
           <Typography
-            fontSize={18}
+            fontSize={{ xs: '1rem', md: '1.125rem' }}
             fontWeight={400}
             color={palette.text.secondary}
           >
             {jobTitle}
           </Typography>
-          <Box mt={2}>
-            <BlocksRendererClient content={description} />
-          </Box>
+          {!isSmallScreen && description && (
+            <Box mt={2}>
+              <BlocksRendererClient content={description} />
+            </Box>
+          )}
         </>
       )}
     </Box>
   );
 };
 
-const SpeakerPreview = ({ compactMode, ...speaker }: SpeakerPreviewProps) => {
-  const flexDirection = compactMode ? 'row' : { xs: 'column', md: 'row' };
-
+const SpeakerPreview = ({
+  compactMode,
+  flexDirection = 'column',
+  ...speaker
+}: SpeakerPreviewProps) => {
+  const isSmallScreen = useMediaQuery('(max-width: 900px)');
   return (
     <Box
       sx={{
+        width: compactMode
+          ? '15.625rem'
+          : isSmallScreen
+          ? '19.5rem'
+          : '16.563rem',
         display: 'flex',
-        flexDirection,
+        flexDirection: { sm: 'row', md: flexDirection },
```

Comment:
you could use a space-between with an inner div with row to have the items aligned as the figma file